### PR TITLE
Documentation Improvement

### LINKED ISSUE #5: Improvements needed
Terrible documentation. Need to improve that with better documentation for all possiblities.
Would love help if someone would like to contribute.

### PR DESCRIPTION
**Pull request by 2kofawsome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,22 +11,711 @@
...
+  }
+], 
+"paging": {'cursors': {}}
+}
+```
+
+
+## Credits
+- [Tony Benoy](https://github.com/tonybenoy)
+- [All Contributors](../../contributors)
```
Comment (tonybenoy) [inline README.md:801]: Nice :+1:

File: README.md [hunk 2]
```diff
@@ -11,22 +11,711 @@
...
 </p>
+
 # ClashOfClansAPI </br>
 Python Wrapper for SuperCells Clash Of Clans API </br>
 Get Token from https://developer.clashofclans.com/</br>
 
-# Install using pip </br>
+# Install
 
 > pip3 install cocapi
```
Comment (tonybenoy) [inline README.md:22]: @2kofawsome do you think we should add a `python setup.py install` or `pip install <PACKAGE-NAME>` ?

File: README.md [hunk 3]
```diff
@@ -11,22 +11,711 @@
...
+
+
+## Clans
+
+### Information about a Clan
+```python
+api.clan_tag(tag) #example tag "#9UOVJJ9J"
+```
+```text
+{
```
Comment (tonybenoy) [inline README.md:52]: @2kofawsome  https://github.com/dear-github/dear-github/issues/166#issuecomment-236342209 can we do a folding here and below for the output? Would make it more readable and beautiful!

File: README.md [hunk 4]
```diff
@@ -11,22 +11,711 @@
         <img src="https://img.shields.io/badge/code%20style-black-000000.svg" alt="Codestyle Black" height="17">
     </a>
 </p>
+
 # ClashOfClansAPI </br>
 Python Wrapper for SuperCells Clash Of Clans API </br>
 Get Token from https://developer.clashofclans.com/</br>
```
Comment (tonybenoy) [inline README.md:18]: @2kofawsome  let's add a link to the doc

**Review by tonybenoy — CHANGES_REQUESTED** [review]
Awesome work @2kofawsome ! Thanks for the help!

**Comment by tonybenoy** [comment]
Solves #5 !

File: README.md (hunk 2 above)
Comment (2kofawsome) [reply README.md:22]: I don't think I've ever used `python setup.py install`  so I would not want to add documentation about using it. I think having `> pip3 install cocapi` is enough, but you may want to add the instructions for `python setup.py install`

File: README.md (hunk 3 above)
Comment (2kofawsome) [reply README.md:52]: Folding is very cool, I will definitely use that again

File: README.md (hunk 2 above)
Comment (tonybenoy) [reply README.md:22]: Cool

**Review by tonybenoy — APPROVED** [review]

**Comment by tonybenoy** [comment]
Thanks again @2kofawsome !

**Pull request merged**